docs/configuration/overview.mdx

```diff
@@ -259,6 +261,38 @@ command:
     workspaceChangelog: true
 ```
 
+### `command/version/changelogs`
+
+Configure aggregate changelogs which document the changes made to multiple
+packages.
+
+```yaml
+command:
+  version:
+    changelogs:
+      - path: FOO_CHANGELOG.md
+        scope: foo_*
+        description: |
+          All notable changes to foo packages will be documented in this file.
+```
+
+### `command/version/changelogs/path`
+
+The path to the changelog file relative to the workspace root.
+
+### `command/version/changelogs/scope`
```

Comment:
Do you think maybe we should have the same syntax as `select-package` in melos run scripts

Will allow all filter types then rather than just scope,

Should just be able to reuse most of the logic to do so?

Thoughts @blaugold?

Reply:
Good point! I think we should do that.